### PR TITLE
[NUI] Change ColorDepth enum to ColorFormat

### PR DESCRIPTION
**Pull request by dkdk-ryu**
ColorDepth was changed to ColorFormat in Dali.

### API Changes ###

 - ACR:

### CODE REVIEW

**Review by tscholb — APPROVED** [review]
LGTM:)

File: src/Tizen.NUI/src/public/Images/NativeImageQueue.cs [hunk 1]
```diff
@@ -42,15 +42,34 @@ public class NativeImageQueue : NativeImageInterface
     {
         private IntPtr handle;
 
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public enum ColorFormat
+        {
+            /// <summary>
+            /// 8 red bits, 8 green bits, 8 blue bits
+            /// </summary>
+            RGB888,
```
Comment (sparrow74) [inline src/Tizen.NUI/src/public/Images/NativeImageQueue.cs:51]: Could you consider the Pascal coding style? A few days ago, I met the similar issue in my patch.
You will change from RGBA8888 to Rgba8888.
I recommend the other cases in TizenFX, as ElmSharp.

**Review by bshsqa — APPROVED** [review]

File: src/Tizen.NUI/src/public/Images/NativeImageQueue.cs (hunk 1 above)
Comment (dkdk-ryu) [reply src/Tizen.NUI/src/public/Images/NativeImageQueue.cs:51]: I referred to PIxelFormat.cs. I think it's the color format standard in NUI.
Should I change it to the pascal style?

Comment (huiyueun) [reply src/Tizen.NUI/src/public/Images/NativeImageQueue.cs:51]: 아래는 C# 레퍼런스의 예제 링크코드입니다. 
https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1712

이 문제에 대한 예제는 아닌데, 예제코드에서 RGB의 경우 RGB형태로 그대로 사용하고 있는걸 볼 수 있어요.

**Review by dongsug-song — APPROVED** [review]
looks good to me. 
according to this page (https://docs.microsoft.com/ko-kr/visualstudio/code-quality/ca1709?view=vs-2019), it would be `Rgba8888`, but in NUI `RGBA8888` looks more intuitive and is used many times, so `RGBA8888` will be OK.

**Pull request merged**